tools/schema-explorer/static/css/style.css

```
@@ -0,0 +1,1077 @@
+   the same line even when the cell is narrow — without this, a tight Key
+   column can wrap the chevron above the key (looks broken). The leaf code
+   itself still allows long names to wrap inside the cell when needed. */
+.schema-tree-indent {
+  display: inline-flex;
+  align-items: baseline;
+  gap: 0.15rem;
+  max-width: 100%;
+}
+.schema-tree-row td.px-3 {
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)